client/src/components/Chat.jsx

```diff
@@ -594,3 +614,7 @@ const Chat = () => {
 };
 
 export default Chat;
+
+Chat.propTypes = {
```

Comment:
Remove typing

Reply:
I am not sure then how to handle typing. because it could be that user came back and is typing but we send a notification regarding his inactivity. would defeat the purpose. other issues i am fixing

Reply:
The one usecase I'm thinking of is when the user wants to type but clears it out or stops because of something.

Now they will never get the notification, see my point?

Reply:
yeah, I see that but what I thought was the person has gone afk or not looking on the screen. That's when the notification should come right? If the person is doing what you just said that doesn't mean he is inactive. is there something else we could do to handle this?

Reply:
I think there's a misunderstanding with this issue, this isn't an offline/online issue, it's a "when a did you text last"